lib/generators/the_policy_machine/add_initial_policy_machine_tables_generator.rb

```diff
@@ -0,0 +1,12 @@
+module ThePolicyMachine
+  module Generators
+    class AddInitialPolicyMachineTablesGenerator < Rails::Generators::Base
+      source_root File.expand_path('../../../migrations', __FILE__)
+
+      def generate_add_cross_assignments_table_migration
```

Comment:
is this method name supposed to be `generate_add_initial_policy_machine_table_migration`?